.eleventy.js

```diff
@@ -61,6 +61,14 @@ module.exports = (eleventyConfig) => {
 		return `${format(new Date(date), 'MMM d, yyyy p')} PST`;
 	});
 
+	eleventyConfig.addFilter('asDateTime', function (date) {
```

Comment:
This appears to be a duplicate of the filter just above it.

Reply:
ooooh. i think that might have been a bad merge on my part